fabcar/javascript/server.js

```diff
@@ -558,4 +557,34 @@ app.post('/api/login', function(req,res){
     main();
 
 })
+
+
+app.post('/api/encryptAndUpload', function(req, res){
+    
+
+    async function main() {
+
+        let fileUpload = require('./encryptAndUpload.js');
+
+        res.status(200).json({Description: 'Encrypted'});
+        
+    }  
+    main();
+    
+})
+
+app.post('/api/downloadAndDecrypt', function(req, res){
```

Comment:
Need to change this to get not post, since we are not inserting or uploading anything
```suggestion
app.get('/api/downloadAndDecrypt', function(req, res){
```